rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java

```diff
@@ -1,22 +1,18 @@
 package org.rewardquest.com.pages;
 
-/*
- * Bahubali P R
- */
-import java.util.List;
-import org.openqa.selenium.By;
-import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.How;
 import org.openqa.selenium.support.PageFactory;
+import org.rewardquest.com.wait.WebDriverWaits;
+import org.testng.Assert;
 
 public class LoginPage {
 
-	public WebDriver driver = null;
-
+	private WebDriver driver = null;
+	private WebDriverWaits wait = null;
+	String textError;

```

Comment:
Why no access modifier for "textError" variable ?

Reply:
Added access modifier for errorText variable.